at_lookup/lib/src/connection/outbound_sync_message_listener.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:at_lookup/src/connection/outbound_message_listener.dart';
+
+class SyncMessageListener extends OutboundMessageListener {
+  Queue pendingData = Queue<List<int>>();
+  var atConst = '@'.codeUnitAt(0);
```

Comment:
this is const..not a var

Reply:
Replaced with a final variable. Getting compile issue when having it as constant.